config/default/kustomization.yaml

```
@@ -30,38 +30,24 @@ bases:
 #- ../prometheus
 
 patchesStrategicMerge:
-# Protect the /metrics endpoint by putting it behind auth.
-# If you want your controller-manager to expose the /metrics
-# endpoint w/o any authn/z, please comment the following line.
-#- manager_auth_proxy_patch.yaml
-
-# Mount the controller config file for loading manager configurations
-# through a ComponentConfig type
-#- manager_config_patch.yaml
-
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
-# crd/kustomization.yaml
-#- manager_webhook_patch.yaml
-
-# [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'.
-# Uncomment 'CERTMANAGER' sections in crd/kustomization.yaml to enable the CA injection in the admission webhooks.
-# 'CERTMANAGER' needs to be enabled to use ca injection
-#- webhookcainjection_patch.yaml
+###################
+# Required patches
+###################
 
 # Enables assisted-service startup to create a route for the service
 # through an init container. This used by in-cluster deployments so that 
 # a route to the service is created and the SERVICE_BASE_URL is updated 
 # to match the route's URL.
 - assisted-service-patch-init-containers.yaml
+
 # An init container that creates a random password and secret for postgres
 # if a secret does not already exist.
 - postgres-patch-init-container.yaml
 
-# Uncomment to set a mininum disk size allowed by the hardware validator.
-# By default the minimum disk size allowed is 120GB. The patch sets the
-# minimum disk size to 20GB.
-#- assisted-service-configmap-patch-hw-validator-min-disk-size.yaml
+# Sets ISO_IMAGE_TYPE to minimal-iso 
+- assisted-service-patch-iso-image-type.yaml
```

```suggestion
- assisted-service-patch-iso-image-use-minimal.yaml
```

It's a nit, but I wonder if using this filename would make more evident what the patching does.